src/betamax/configure.py

```diff
@@ -19,6 +18,7 @@ class Configuration(object):
 
     """
 
+    _Betamax_class = None  # prevent circular import
```

Comment:
Since Configuration handles the global configuration that I was silly enough to think was a good idea, and we [create a copy](https://github.com/betamaxpy/betamax/blob/d3f4bd89b2b56e11e0b03b6eb08c2b48727425fc/src/betamax/recorder.py#L57) for each Recorder, can we simplify this by storing the hooks on this class?

Then we can alter `dispatch_hooks` in the `recorder` file as I'll comment down there.